File: justmeetbackend/src/main/java/it/unicam/justmeetbackend/databasefake/Topic.java
package it.unicam.justmeetbackend.databasefake;

import org.springframework.data.annotation.Id;

public class Topic {

    @Id
    private int _id;
    
    private String argomento;

    public Topic(){};

    public Topic(int id, String a){
        this._id = id;
        this.argomento = a;
    }

    /**
     * @return the argomento
     */
    public String getArgomento() {
        return argomento;
    }

    /**
     * @param argomento the argomento to set
     */
    public void setArgomento(String argomento) {
        this.argomento = argomento;
    }

    public int get_id() { return _id; }

    public void setId(int id) {
        this._id = id;
    }

}

File: justmeetfrontend/lib/classi/topic.dart


class Topic{

   String argomento;
   int _id;

	

  
  Topic(this._id, this.argomento);

  Topic.fromjson(Map<String, dynamic> json)
  : 
  _id = json['_id'],
  argomento = json['argomento'];

  int getId() {
		return this._id;
	}

	void setId(int _id) {
		this._id = _id;
	}
  
}